.github/workflows/release.yml

```diff
@@ -32,16 +33,17 @@ jobs:
         uses: subosito/flutter-action@v2
         with:
           channel: 'stable'
+          cache: true
       - name: Install Flutter dependencies
         run: flutter pub get
       - name: Generate files with Builder
         run: flutter packages pub run build_runner build --delete-conflicting-outputs
       - name: Build with Flutter
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SIGNING_KEY_ALIAS: ${{ secrets.SIGNING_KEY_ALIAS }}
-          SIGNING_KEY_PASSWORD: ${{ secrets.SIGNING_KEY_PASSWORD }}
-          SIGNING_STORE_PASSWORD: ${{ secrets.SIGNING_KEYSTORE_PASSWORD }}
+          #SIGNING_KEY_ALIAS: ${{ secrets.SIGNING_KEY_ALIAS }}

```

Comment:
Does this step not require the keystore credentials?

Reply:
Whoops, the revert might have lost during the squash, I'll go ahead and fix it